src/Multiple/SmallMultiplePlainTextFormatter.php

```diff
@@ -4,25 +4,78 @@
 
 namespace CultuurNet\CalendarSummaryV3\Multiple;
 
+use CultuurNet\CalendarSummaryV3\DateComparison;
+use CultuurNet\CalendarSummaryV3\DateFormatter;
 use CultuurNet\CalendarSummaryV3\Offer\Offer;
-use CultuurNet\CalendarSummaryV3\Periodic\MediumPeriodicPlainTextFormatter;
+use CultuurNet\CalendarSummaryV3\PlainTextSummaryBuilder;
 use CultuurNet\CalendarSummaryV3\Translator;
+use DateTimeZone;
 
 final class SmallMultiplePlainTextFormatter implements MultipleFormatterInterface
 {
+    /**
+     * @var DateFormatter
+     */
+    private $formatter;
+
     /**
      * @var Translator
      */
     private $translator;
 
     public function __construct(Translator $translator)
     {
+        $this->formatter = new DateFormatter($translator->getLocale());
         $this->translator = $translator;
     }
 
     public function format(Offer $offer): string
     {
-        $formatter = new MediumPeriodicPlainTextFormatter($this->translator);
-        return $formatter->format($offer);
+        $startDate = $offer->getStartDate()->setTimezone(new DateTimeZone(date_default_timezone_get()));
+        $endDate = $offer->getEndDate()->setTimezone(new DateTimeZone(date_default_timezone_get()));
+
+        if (DateComparison::onSameDay($startDate, $endDate)) {
+            if (DateComparison::isThisEvening($startDate)) {
+                return $this->translator->translate('tonight');
+            }
+            if (DateComparison::isToday($startDate)) {
+                return $this->translator->translate('today');
+            }
+            if (DateComparison::isTomorrow($startDate)) {
+                return $this->translator->translate('tomorrow');
+            }
+            if (DateComparison::isCurrentWeek($startDate)) {
+                $preposition = $this->translator->translate('this');
+                $weekDay = $this->formatter->formatAsDayOfWeek($startDate);
+                return PlainTextSummaryBuilder::singleLine($preposition, $weekDay);
+            }
+            $plainTextSummaryBuilder = PlainTextSummaryBuilder::start($this->translator)
+                ->append($this->formatter->formatAsAbbreviatedDayOfWeek($startDate))
+                ->append($this->formatter->formatAsDayNumber($startDate))
+                ->append($this->formatter->formatAsAbbreviatedMonthName($startDate));
+            if (!DateComparison::isCurrentYear($startDate)) {
```

Comment:
I did not find any unit tests where the current year had to be removed.
So I did not find anything like `22 okt - 13 mei 2023` or `22 okt 2021 - 13 mei`

This seems to be missing from other tests as well.